### PR TITLE
Feature/themes

### LINKED ISSUE #21: Themes
Add more themes and add option to select themes
```
fman --theme dracula
```

### PR DESCRIPTION
**Pull request by dzzzchhh**
### Topic
Proposal for handling themes.

### Notable changes
- Introduces CLI arguments handling with help of [go-arg](https://github.com/alexflint/go-arg)
- Adds a brogrammer theme option (in its current state it only resembles the brogrammer palette, I added it to showcase the ability of switching between themes)
- If no `--theme` option is provided then default theme is picked and displayed.

### Other changes
- Sorry for removing the `updatePath` function in `main.go` - it was the only way to make CLI arguments work and not break the filepaths. Everything seems to work properly, but please do tell if I made a mistake by removing it, I'm not too efficient with `bubbletea`.

Closes #21 

https://user-images.githubusercontent.com/8414298/194067786-6c24f52a-3287-41a4-b5b0-1d871ec50f90.mp4

### CODE REVIEW

**Review by nore-dev — CHANGES_REQUESTED** [review]
Thank you @dzzzchhh

File: theme/config.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package theme
+
+type ThemeMap map[string]Theme
+
+var themes = ThemeMap{
+	"default":    DefaultTheme,
+	"darcula":    DefaultTheme,
```
[the remark this answers is not in the document]
Comment (dzzzchhh) [reply theme/config.go]: Yes, the canonical name is indeed `Dracula`

File: main.go [hunk 2]
```diff
@@ -94,14 +81,21 @@ func (app *App) View() string {
 	))
 }
 
+// Define CLI arguments
+var args struct {
+	Theme string `default:"default"`
+}
+
```
[the remark this answers is not in the document]
Comment (dzzzchhh) [reply main.go]: Done, very nice suggestion

File: main.go [hunk 3]
```diff
@@ -24,20 +24,7 @@ type App struct {
...
 func (app *App) Init() tea.Cmd {
-	return tea.Batch(app.infobarModel.Init(), app.UpdatePath(), app.listModel.Init())
-}
-
-func (app *App) UpdatePath() tea.Cmd {
-	return func() tea.Msg {
-		path := "."
-
-		if len(os.Args) >= 2 {
-			path = os.Args[1]
```
[the remark this answers is not in the document]
Comment (dzzzchhh) [reply main.go:35]: Done

File: main.go [hunk 4]
```diff
@@ -24,16 +25,12 @@ type App struct {
 }
 
 func (app *App) Init() tea.Cmd {
-	return tea.Batch(app.infobarModel.Init(), app.UpdatePath(), app.listModel.Init())
+	return tea.Batch(app.infobarModel.Init(), app.listModel.Init())
```
Comment (nore-dev) [inline main.go:28]:
```suggestion
	return tea.Batch(app.infobarModel.Init(), app.UpdatePath(), app.listModel.Init())
```

**Review by nore-dev — CHANGES_REQUESTED** [review]

**Review by nore-dev — APPROVED** [review]

**Comment by nore-dev** [comment]
thank you again @dzzzchhh

**Comment by nore-dev** [comment]
i forgot to squash again

**Pull request merged**